autogpt_platform/frontend/src/app/library/agents/[id]/page.tsx

```diff
@@ -219,7 +220,7 @@ export default function AgentRunsPage(): React.ReactElement {
 
   if (!agent || !graph) {
     /* TODO: implement loading indicators / skeleton page */
-    return <span>Loading...</span>;
+    return <LibraryRunLoadingSkeleton />;
```

Comment:
https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming

should we just be using suspense though

Reply:
oh shoot didn't realize this was client side. that's wild